.gitignore

```diff
@@ -2,3 +2,8 @@
 # External/*
 /test-suite-externals
 *.pyc
+build/
+Makefile.config
+config.log
+config.status
+mklib
```

Comment:
I added it just not to upload some build file. If these files are necessary, I'll delete them from .gitignore.

Reply:
But I think, that I accidentally put .gitignore to this PR

Reply:
You don't have to delete all of .gitignore. I am not saying the additions to the .gitignore is necessarily a bad idea, just that it should be a separate PR so it can be reviewed independenty of the new test.

Reply:
OK, I got your point.

Reply:
> You don't have to delete all of .gitignore.
> I got your point.

But you "deleted all of .gitignore". :)

Reply:
I thought it was a good option not to put it in this PR, because it wouldn't affect on workability at all. :)

Reply:
`it`? What Steffen said is - do not make any to `.gitignore` in this PR. Currently your PR removes `.gitignore` file. I don't think we should do this, but if you really want to make changes to this file, please, make them in a separate PR.
Please, recover original content of the `.gitignore` file.

Reply:
I recovered original `.gitignore` file. Thanks for clarification.

Reply:
Thank you!